sbnobj/Common/Calibration/TrackCaloSkimmerObj.h

```diff
@@ -98,6 +101,42 @@ namespace sbn {
      iproc(-1) {}
   };
 
+  struct TrueHit {
+    int16_t cryo; //!< Cryostat of hit
+    int16_t tpc; //!< TPC of hit
+    int16_t plane; //!< Plane of hit
+    int16_t wire; //!< Wire of hit
+    int16_t channel; //!< Channel of hit
+
+    unsigned ndep; //!< Number of depositions in hit
+    float nelec; //!< Number of electrons in hit
+    float e; //!< energy in hit [MeV]
+    float pitch; //!< Track pitch for hit [cm]
+    float rr; //!< Track residual range for hit [cm]
+    int itraj; //!< Index of hit along trajectory
+    Vector3D p; //!< Location of hit [cm]
+    float time; //!< Time of hit [ticks]
+
+    TrueHit():
+      cryo(-1),
+      tpc(-1),
+      wire(-1),
+      channel(-1),
+      ndep(0),
+      nelec(0.),
+      e(0.),
+      pitch(0.),
+      rr(0.),
+      itraj(-1),
+      time(0.)
+      {
+        // set the location to 0.
+        p.x = 0;
+        p.y = 0;
+        p.z = 0;
+      }
+  };
+
```

Comment:
As from in the previous review iteration, I think this to be an issue important enough for maintainability to insist to the C++11-style initialization pattern.
Let's discuss this in a meeting, then.

Reply:
@PetrilloAtWork -- if this needs a longer conversation. Can we merge in this PR and fix it if necessary down the line?